Gemfile

```diff
@@ -8,8 +8,6 @@ gem 'pg'
 gem 'sass-rails'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
-# Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.2.0'
```

Comment:
As far as I can tell, this wasn't being used and throws this warning:

```
DEPRECATION WARNING: Single arity template handlers are deprecated. Template handlers must
now accept two parameters, the view object and the source for the view object.
Change:
  >> Coffee::Rails::TemplateHandler.call(template)
To:
  >> Coffee::Rails::TemplateHandler.call(template, source)
 (called from <top (required)> at /Users/jesseproudman/Source/betterment/test_track/config/environment.rb:5)
```

Reply:
Yup. We never used coffee script in this app. So we should have removed it sooner.